src/components/organisms/ExplorerPane/HelmPane/HelmChartRenderer/HelmChartRenderer.tsx

```diff
@@ -43,6 +44,17 @@ const HelmChartRenderer: React.FC<IProps> = props => {
 
   const [isHovered, setIsHovered] = useState<boolean>(false);
 
+  const chartName = useMemo(() => {
+    let name = helmChart.name;
+    let parentChart = helmChart.parentChartId ? helmChartMap[helmChart.parentChartId] : undefined;
+    while (parentChart) {
+      name = `${parentChart.name}/${name}`;
+      parentChart = parentChart.parentChartId ? helmChartMap[parentChart.parentChartId] : undefined;
+    }
+
+    return name;
+  }, [helmChart, helmChartMap, dispatch]);
```

Comment:
Can we remove the `dispatch` unnecessary dependency

Reply:
duh - thanks!